general-superstaq/dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
-black-configparser~=0.2.0b2  # warning: potential issues https://github.com/danie1k/python-black-configparser/blob/5c6c8d122cdce95e2c7b4ada773cf2f4267c9189/README.md#known-issues
 black[jupyter]~=22.3.0
 flake8>=3.8.4
+flake8-pyproject
```

Comment:
Note: imo this should not have any version number attached to it, so that `pip` automatically figures out which version is compatible with `flake8`

Reply:
done